### DISCUSSION TITLE
Refound

### DISCUSSION THREAD

**Discussion by gustavovt** [aclap-dev/video-downloadhelper#1056] [Help and Q&A]
### Your question

I bought the extension without knowing that it doesn't work on Google with Chrome.

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: pt-PT
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"9ed56e9ab0bf4cbe"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 3,
    "successfull_dl": 84
}

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by gustavovt** [comment]
Another purchase appeared on my credit card. Instead of refunding my
purchase, another purchase was made.

Em sex., 26 de jul. de 2024, 3:28 AM, jcfrog ***@***.***>
escreveu:

**Reply by jcfrog** [reply]
That's really strange. We actually don't make the refund, Stripe does. But I never heard about such an issue.
Stripe status from yesterday: "Successfully refunded due to customer request. It may take a few days for the money to reach the customer's bank account."
